src/ReachSets/project_reach.jl

```diff
@@ -120,13 +125,12 @@ This array contains the collection of reach sets in 2D.
 It is assumed that the variable given in vars belongs to the block computed
 in the sequence of 2D sets `Rsets`.
 """
-function project_reach(
+function project_2d_reach(
         Rsets::Vector{<:ReachSet{<:LazySets.LazySet{numeric_type}}},
         vars::Vector{Int64},
         n::Int64,
         options::Options)::Vector{<:ReachSet} where {numeric_type<:Real}
-    # parse input
-    assert(length(vars) == 2)
```

Comment:
Why remove that?

Reply:
Why do we need it 2 times? We already checked previously

Reply:
Oh, right!